src/components/QueryTags/index.tsx

```diff
@@ -26,22 +27,47 @@ const QueryTags: FC<QueryTagsProps> = ({ content, type }) => {
     }
   };
 
+  if (!content) return null;
+
   return (
     <Space size={10}>
-      {content?.map((tag) => {
-        const tagSplited = tag.split(".");
-        return (
-          <NestedTag
-            key={JSON.stringify(tag)}
-            tag={{ title: tagSplited[0], color: QUERY_COLORS[type][0] }}
-            nested={tagSplited.slice(1).map((t, idx) => ({
-              title: detectIcon(t),
-              color: QUERY_COLORS[type][1],
-              key: idx,
-            }))}
-          />
-        );
-      })}
+      {Array.isArray(content)
+        ? content?.map((tag) => {
+            let tagSplited: string[] = [];
+
+            if (typeof tag === "string") {
+              tagSplited = tag.split(".");
+            } else {
+              tagSplited = `${tag.dimension}.${tag.granularity}`.split(".");
```

Comment:
every object key should be validated on presence, undefined objects and errors crash the app, so it's better to set optional chaining instead of getting crashed app.

`tag?.dimension` and `tag?.granularity`